dashboard/src/components/StageCard.tsx

```
@@ -0,0 +1,323 @@
+  List,
+  ListItem,
+  ListItemText,
+  ListItemIcon,
+  Divider,
+  Button,
+} from '@mui/material';
+import {
+  ExpandMore,
+  ExpandLess,
+  CheckCircle,
+  Error,
```

Biome (2.1.2)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)